### ISSUE TITLE
meson-python: The poppler package is split between purelib and platlib

### ISSUE THREAD

**Issue by bzamecnik** [cbrunet/python-poppler#74]
`pip install python_poppler==0.4.0` on python 3.10 (Ubuntu 22.04) using meson 1.1.0 fails with:

```
#30 28.35       + meson install --no-rebuild --destdir /tmp/pip-install-d59qdq37/python-poppler_8100653bafcf4616b4f77887e540edc4/.mesonpy-zrqhtx8x/install
#30 28.35       
#30 28.35       meson-python: error: The poppler package is split between purelib and platlib: 'purelib/poppler/__init__.py' and 'platlib/poppler/cpp/global_.cpython-310-x86_64-linux-gnu.so', a "pure: false" argument may be missing in meson.build
#30 28.35       [end of output]
```

-> `"pure: false"` argument may be missing in meson.build

Caused by [meson-python-0.13.0 release](https://github.com/mesonbuild/meson-python/releases/tag/0.13.0):
> - Raise an error when a package is split between ``platlib`` and ``purelib``.

More info: https://github.com/cbrunet/python-poppler/issues/74

I'd guess that we need to pass `pure: false` in the `meson.build` file here (similar to [test code of meson](https://github.com/mesonbuild/meson/pull/10783/files#diff-4db40f1cbca91510c6998c4db81dc06a871246d734c221ed3979eddce0a5e05fR14), [release notes](https://mesonbuild.com/Release-notes-for-0-64-0.html#pythonfind_installation-now-accepts-pure-argument
), [docs](https://mesonbuild.com/Python-module.html#find_installation)):
```
python3 = python_mod.find_installation('python3', pure: false)
```

**Comment by frivas-at-navteca** [comment]
I am having the same issue, it is really awesome there is a fix already. I am wondering when it will be released so the package installed via pip will work.

Thanks for fixing it so fast!

**Comment by bzamecnik** [comment]
Yeah, it broke a build and needed to be fixed ASAP.

**Comment by eRuaro** [comment]
Hi @bzamecnik, any idea how to circumvent this issue when trying to install poppler on docker? Here's my dockerfile, and my requirements.txt
```
FROM python:3.10-slim-buster

# Update package lists
RUN apt-get update && apt-get install ffmpeg libsm6 libxext6 gcc g++ git build-essential libpoppler-cpp-dev pkg-config poppler-utils tesseract-ocr libtesseract-dev -y

# Make working directories
RUN  mkdir -p  /app
WORKDIR  /app

# Copy the requirements.txt file to the container
COPY requirements.txt .

# Install dependencies
RUN pip install --upgrade pip

RUN pip install torch torchvision torchaudio

RUN pip install -r requirements.txt

RUN pip install 'git+https://github.com/facebookresearch/detectron2.git@v0.4#egg=detectron2'

# Copy the .env file to the container
COPY .env .

# Copy every file in the source folder to the created working directory
COPY  . .

# Expose the port that the application will run on
EXPOSE 8000

# Start the application
CMD ["python", "-m", "uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000"]
```

```
fastapi
uvicorn
langchain
python-poppler
pytesseract
unstructured[local-inference]
psycopg2-binary
pgvector
openai
tiktoken
python-dotenv
```

**Comment by frivas-at-navteca** [comment]
Hello @eRuaro 

Great question. I hope this answer helps you somehow. 

In my specific case I did this 
```
FROM public.ecr.aws/lambda/python:3.9
...
RUN pip3 install 'git+https://github.com/facebookresearch/detectron2.git'
...
RUN yum install tesseract tesseract-langpack* poppler poppler-cpp poppler-utils -y
RUN pip3 install --no-cache-dir --upgrade deepdoctection[pt]
RUN pip3 install 'git+https://github.com/mindee/doctr.git'
```

My suggestion would be trying installing from the source, something like this:
```
RUN pip3 install 'git+https://github.com/cbrunet/python-poppler.git'
```

I can see that the main already includes [the fix](https://github.com/cbrunet/python-poppler/commit/92c66ac7bab601d03e71a4c9e5a08f17872361d6) so I would think the new package will be ready in short.

**Comment by cbrunet** [comment]
Fixed by #75
